F1WM/ApiModel/CalendarRace.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using F1WM.DatabaseModel;
+
+namespace F1WM.ApiModel
+{
+    public class CalendarRace
+    {
+        [DisplayFormat(DataFormatString = "{yyyy/mm/dd:HH:MM:ss}")]
+        public DateTime date { get; set; }
+        public QualifyingResultPosition polePositionLapResult { get; set; }
```

Comment:
The type is probably wrong here. It should be LapResultSummary instead, like in NextRaceSummary for example.